app/cells/date_range/date_range.coffee

```diff
@@ -30,17 +30,21 @@ $(document).on 'uic:domchange', (e) ->
     $start_input = $($el.data().start)
     $end_input = $($el.data().end)
 
+
+
     start_date = _.find [$el.data('startDate'), $start_input.val(), yesterday],
                         (val) -> val && val.toString().length > 0
     end_date = _.find [$el.data('endDate'), $end_input.val(), yesterday],
                       (val) -> val && val.toString().length > 0
 
     options = _.extend({},
       defaults,
-      _.pick($el.data(), ['dateLimit', 'ranges', 'submitOnChange', 'opens']),
+      _.pick($el.data(), ['dateLimit', 'ranges', 'submitOnChange', 'opens', 'today']),
       { startDate: start_date, endDate: end_date })
 
     options.ranges = _.mapObject(options.ranges, (v, k) -> _.map(v, (v) -> moment(v)))
+    options.ranges = _.extend({}, {'Today': [today(), today()]}, options.ranges) if options.today
```

Comment:
Looks like "Yesterday" gets defaulted up on lines 33 & 35 where the start and end dates are initialized.  I think that's how it works, anyway.